.mergify.yml

```diff
@@ -37,15 +37,12 @@ queue_rules:
               - "status-success=ci/centos/k8s-e2e-external-storage/1.27"
               - "status-success=ci/centos/k8s-e2e-external-storage/1.28"
               - "status-success=ci/centos/k8s-e2e-external-storage/1.29"
-              - "status-success=ci/centos/k8s-e2e-external-storage/1.30"
```

Comment:
There's one more set here https://github.com/ceph/ceph-csi/blob/c96e05a4c257ebafbf587a9f7f0ac0f991585cb4/.mergify.yml#L268-L294

Reply:
instead of this one i would like to run 1.30 test as we might need to do minor releases due to bug fixes.

Reply:
okay, then I'll include the 1.30 tests for `release-v3.11`